db/schema.rb

```diff
@@ -19,9 +19,15 @@
   create_table "accepted_tax_return_analytics", force: :cascade do |t|
     t.bigint "advance_ctc_amount_cents"
     t.datetime "created_at", null: false
+    t.bigint "ctc_amount_cents"
+    t.bigint "eip1_and_eip2_amount_cents"
     t.bigint "eip3_amount_cents"
-    t.bigint "refund_amount_cents"
+    t.bigint "eip3_amount_received_cents"
+    t.bigint "outstanding_ctc_amount_cents"
+    t.bigint "outstanding_eip3_amount_cents"
     t.bigint "tax_return_id"
```

Comment:
since there is a relationship to tax_return which indicates the year, we may not actually need to explicitly store the tax return year. 🤔  data team should be able to query for only the accepted analytics using a join to the tax return year which already exists.

We also might want to consider adding an index on tax return on year OR on this table for tax_return_year if we keep it, since data team will likely be querying based on year pretty often and an index can help that query's performance.